controller_manager:
  ros__parameters:
    update_rate: 50  # Hz

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


    right_front_controller:
      type: velocity_controllers/JointGroupVelocityController

    left_front_controller:
      type: velocity_controllers/JointGroupVelocityController

    right_rear_controller:
      type: velocity_controllers/JointGroupVelocityController

    left_rear_controller:
      type: velocity_controllers/JointGroupVelocityController



right_front_controller:
  ros__parameters:
    joints:
      - right_front_wheel_joint
    pid: { p: 10.0, i: 0.01, d: 0.0 }
    publish_rate: 80.0

left_front_controller:
  ros__parameters:
    joints:
      - left_front_wheel_joint
    pid: { p: 10.0, i: 0.01, d: 0.0 }
    publish_rate: 80.0

right_rear_controller:
  ros__parameters:
    joints:
      - right_rear_wheel_joint
    pid: { p: 10.0, i: 0.01, d: 0.0 }
    publish_rate: 80.0

left_rear_controller:
  ros__parameters:
    joints:
      - left_rear_wheel_joint
    pid: { p: 10.0, i: 0.01, d: 0.0 }
    publish_rate: 80.0

